catalog/app/components/Filters/NumbersRange.tsx

```diff
@@ -1,17 +1,71 @@
 import * as React from 'react'
 import * as M from '@material-ui/core'
+import * as d3Scale from 'd3-scale'
 
 import * as Notifications from 'containers/Notifications'
 import { formatQuantity } from 'utils/string'
 
 const isNumber = (v: unknown): v is number => typeof v === 'number' && !Number.isNaN(v)
 
-const valueLabelFormat = (number: number) =>
-  // @ts-expect-error
-  formatQuantity(number, {
-    suffixes: ['', 'K', 'M', 'B', 'T', 'P', 'E', 'Z', 'Y'],
+export function createNonLinearScale({ min, max }: { min: number; max: number }): {
+  marks: M.Mark[]
+  scale: d3Scale.ScalePower<number, number>
+} {
+  const scale = d3Scale.scalePow().exponent(4).domain([0, 100]).range([min, max])
+  const marks: M.Mark[] = scale
+    .ticks(101) // from 0 to 100, including 0
+    .map((value) => ({ value: Math.round(scale(value)) }))
+  return { marks, scale }
+}
+
+const ExpSuffix = ({ n }: { n: number }) => (
+  <>
+    ×10<sup>{n}</sup>
+  </>
+)
+
+const createValueLabelFormat =
+  (scale: d3Scale.ScalePower<number, number>) => (number: number) =>
+    formatQuantity(Math.round(scale(number)), {
+      suffixes: [
```

Comment:
default suffixes seem more suitable here, because at least they fit somewhat better into the "bubble"